qutip/qip/circuit_latex.py

```diff
@@ -59,6 +60,9 @@ def _latex_compile(code, filename="qcirc", format="png"):
     if format == 'png':
         os.system("convert -density %s %s.pdf %s.png" % (100, filename,
                                                          filename))
+        if sys.platform == 'win32':
+            os.system("magick convert -density %s %s.pdf %s.png"
+                      % (100, filename, filename))
```

Comment:
This duplicates the system command if the platform is Windows (and also duplicates the code).  Perhaps it would be better to do something like `command  = magick_command + magick_arguments.format(100, filename, filename)`, and detect/set `magick_command` only once (could be cached)?

Reply:
From your response, I searched a little about ImageMagick. 
As you pointed out, command 'convert' was replaced with 'magick' from version 7 or higher.
But I'm not sure whether this works well in Mac. According to your response, 'convert' seems to still work well in Mac. Could you check 'magick' command also works in Mac? If it works, it would be better to adopt the command 'magick'. And if it does not work, could you check your ImagaMagick's version? Sorry to bother you. I want to check by myself, but I don't have Mac.

Reply:
Edit: Both `convert` and `magick` work for me, on Mac 10.14.5, ImageMagick 7.0.8-49 Q16 x86_64

Reply:
Both `convert` and `magick` work for me with ImageMagick installed by homebrew, version `7.0.9-16`.

While this is beside the point of the PR, perhaps you're missing a symlink to `magick`, Nathan?  My install has both the `magick` and `convert` binaries (although they're identical...).

Reply:
I think my problem depends on OS. As far as I know command 'convert' is already native command in Windows ( it is used to change disk type: completely different purpose). In my guess, ImageMagick's changing command is reflecting this fact.